Cli-CredentialHelper/Installer.cs

```diff
@@ -93,6 +93,7 @@ private static string UserBinPath
                 return _userBinPath;
             }
         }
+
```

Comment:
We try yo keep `public <type> Accessor` and `private <type> _accessor` immediately next to each other; ideally without the extra vertical white space.